test/advanced.test.js

```diff
@@ -277,6 +287,21 @@ describe("mithril-objectify", function() {
                 '({tag:"div",attrs:undefined,children:[JSON.stringify({})],dom:undefined,domSize:undefined,events:undefined,key:undefined,state:{},text:undefined});'
             );
         });
+
+        it("should not convert JSON when entries are not objects", function() {
```

Comment:
Sorry, I actually meant to make a separate PR with this... (Partly because I was still checking if it's necessary.)

(Having said that, I _think_ it is? Using Mithril 1.x there's an exception when using mopt with `m('pre', m('code', { class: classNames(vnode.attrs.class) }, JSON.stringify(vnode.attrs.node, null, 2)))`; I think it's getting into `createComponent` (via createNode) with the string value, and then the `assign(vnode.state, vnode.tag)` fails.)